### ISSUE TITLE
False positive for is_resource checking for closed resources

### ISSUE THREAD

**Issue by amorimjuliana** [phpstan/phpstan-strict-rules#60]
The function `is_resource` checks for an open resource:

```
Note:
is_resource() is not a strict type-checking method: it will return FALSE if var is a resource variable that has been closed.
```

ttps://phpstan.org/r/38a02adb-d114-4636-adab-69ecd2cdce76
